tokio/src/sync/mpsc/unbounded.rs

```diff
@@ -223,6 +223,22 @@ impl<T> UnboundedReceiver<T> {
         self.chan.close();
     }
 
+    /// Checks if the channel has been closed. This happens when the
+    /// [`UnboundedReceiver::close`] method is called.
+    ///
+    /// [`UnboundedReceiver::close`]: crate::sync::mpsc::UnboundedReceiver::close
+    ///
```

Comment:
Add an `# Examples` header before the example.

It would probably also be good to add `use tokio::sync::mpsc;` at the top of the example as that's done in **almost** all the other examples (the noteable exception is of course `Sender::is_closed`).